platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -139,17 +139,33 @@ event_processor:
   context_path: /
   # The listening port of the event processing server
   port: 8084
+  # The flag to enable the status callback to the client domain
+  client_domain_status_callback_enabled: true
+  # The flag to enable the event delivery to the anchor business server
+  callback_api_event_delivery_enabled: true
   # The queue where the event processor will
   queue:
     type: kafka
     kafka:
       bootstrap_server: localhost:29092
-      event_type_to_topic:
-        all: dev_ap_event_single_queue
-        quote_created: ap_event_quote_created
-        transaction_created: ap_event_transaction_created
-        transaction_status_updated: ap_event_transaction_status_changed
-        transaction_error: ap_event_transaction_error
+
+# The list of clients for the Anchor server to safely communicate with the outside client domains.
+# Each item in the list may contain the following fields:
+#   - domain: the domain of the client
+#   - type: the type of the client. The supported values are `custodial` and `non-custodial`
+#   - public_key: the public key of the client. This must match the public key of the TOML file of the client domain
+#   - url: the URL of the client's callback API endpoint
+#
+# Examples:
+#     - domain: lobstr.com
+#       type: non-custodial
+#       public_key: GBI2IWJGR4UQPBIKPP6WG76X5PHSD2QTEBGIP6AZ3ZXWV46ZUSGNEGN2
```

Comment:
Does non-custodial wallet even have a public key? I think it should only be required for custodial wallets

Reply:
Doesn't lobstr.com server has a `SIGNING_KEY` in the TOML file?

cc: @JakeUrban

Reply:
Noncustodial wallets do have a public key, published on their stellar.toml file. This is nice because the non-custodial wallet could rotate their signing key and the anchor's authentication process would still work because it should fetch the `SIGNING_KEY` on each authentication request.

For this same reason, I don't think we should require each wallet entry to have a public key attribute. Only custodial wallets will need the `public_key` attribute.

Reply:
I think we can add `type` when there is an immediate use case. In that case, the `public_key` will stay.